examples/examples.ipynb

```diff
@@ -874,15 +846,13 @@
    "source": [
     "### Confusion Matrix \n",
     "\n",
-    "The confusion matrix visualizer displays the accuracy score of the model, i.e., it shows how each of the test values predicted classes compare to their actual classes. It provides the numerical scores as well as a color-coded heatmap to provide data scientists a clearer view of the model performance on each of the individual classes and is particularly useful for imbalanced datasets. More information can be found by looking at the scikit-learn documentation on http://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html."
+    "The `ConfusionMatrix` visualizer displays the accuracy score of the model, i.e., it shows how each of the test values predicted classes compare to their actual classes. It provides the numerical scores as well as a color-coded heatmap to provide data scientists a clearer view of the model performance on each of the individual classes and is particularly useful for imbalanced datasets. More information can be found by looking at the [scikit-learn documentation](http://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html)."
```

Comment:
I only made two changes here to the text in the Jupyter notebook. First, I changed "confusion matrix visualizer" to "`ConfusionMatrix` visualizer" and I embedded the link to the scikit-learn documentation at the end of this paragraph.